### DISCUSSION TITLE
Feature Klackender Mod

### DISCUSSION THREAD

**Discussion by Bersaker** [mriscoc/Ender3V2S1#763] [Ideas]
Hi. Is it possible to add this feature in the configurator? 

The description of what needs to be changed in the configuration is in the description from 

https://github.com/kevinakasam/KlackEnder-Probe

**Comment by mriscoc** [comment]
Just use the files available in this PR: https://github.com/mriscoc/Special_Configurations/pull/24
Download [this file](https://github.com/mriscoc/Special_Configurations/raw/d29436041b311aedb9ebe7246ff16577048e3e59/_leveling/KlackEnder.json) into the _leveling folder

**Reply by Bersaker** [reply]
Thanks.  As soon as I've rebuilt the e3v2 I'll test it and report if it works.
